higher-order-effects.cabal

```diff
@@ -13,23 +13,13 @@ extra-source-files:  ChangeLog.md
 cabal-version:       >=1.10
 
 library
-  exposed-modules:     Control.Carrier
-                     , Control.Carrier.Either
-                     , Control.Carrier.Identity
-                     , Control.Carrier.List
-                     , Control.Carrier.Maybe
-                     , Control.Carrier.Reader
-                     , Control.Carrier.Split
-                     , Control.Carrier.State
-                     , Control.Carrier.Writer
```

Comment:
Carriers are now one-to-one with effect handlers, meaning that we can’t e.g. have two different `TermAlgebra` instances for `StateH`. Thus, we 🔥 the separate carrier modules and instead define the carriers in the effect files alongside the `run*` functions.